canonical/canonical.com
# Wd 4287 dev canonical com blog entry

## Done

Update the look of individual blog articles. I've rebased this on top of the blog listing pr, as it uses shared styles and markup.
Not quite sure what to put in the breadcrumbs - I've put the type of blog post for now, open to other suggestions.
## QA

- Check out this feature branch
- Run the site using the command `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check against the mockup.

## Issue / Card

Fixes #

## Screenshots

![image](https://github.com/canonical/canonical.com/assets/2741678/1b436636-fda8-4705-b87c-9f0c9e796b4f)
